### PR TITLE
feat: created JDBCToJDBC template

### PR DESCRIPTION
**Pull request by poojabasker20**

### CODE REVIEW

File: java/src/main/java/com/google/cloud/dataproc/templates/jdbc/JDBCToJDBC.java [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+  public void runTemplate() {
+    try (SparkSession spark = SparkSession.builder().appName("JDBC to JDBC").getOrCreate()) {
+      // Set log level
+      spark.sparkContext().setLogLevel(config.getSparkLogLevel());
+      Dataset<Row> inputData = load(spark);
+      write(inputData);
+
+      if (StringUtils.isNotBlank(config.getJdbcOutputPrimaryKey())) {
+        addPrimaryKeyColumn();
+      }
```
Comment (vanshaj-bhatia) [inline java/src/main/java/com/google/cloud/dataproc/templates/jdbc/JDBCToJDBC.java:66]: Primary key should only be added if no user transformation has been performed.
`inputData = spark.sql(config.getJdbcTempSQLQuery());`

File: java/src/main/java/com/google/cloud/dataproc/templates/jdbc/JDBCToJDBC.java [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+      statement.executeUpdate(
+          String.format(
+              "ALTER TABLE %s ADD PRIMARY KEY (%s)",
+              config.getJdbcOutputTable(), config.getJdbcOutputPrimaryKey()));
+
+      statement.close();
+      connection.close();
+    } catch (SQLException e) {
+      e.printStackTrace();
+    }
```
Comment (vanshaj-bhatia) [inline java/src/main/java/com/google/cloud/dataproc/templates/jdbc/JDBCToJDBC.java:123]: Avoid try-catch

**Review by vanshaj-bhatia — CHANGES_REQUESTED** [review]

**Comment by vanshaj-bhatia** [comment]
Please update README and java/README files

**Comment by poojabasker20** [comment]
> Please update README and java/README files

Updated

File: java/src/main/java/com/google/cloud/dataproc/templates/jdbc/README.md [hunk 3]
```diff
@@ -266,3 +337,201 @@ There are two optional properties as well with "JDBC to SPANNER" Template. Pleas
...
+
+* MySQL to Microsoft SQL Server
+```
+bin/start.sh \
+-- --template JDBCTOJDBC \
+--templateProperty jdbctojdbc.input.url="jdbc:mysql://1.1.1.1:3306/db-test?user=test&password=password" \
+--templateProperty jdbctojdbc.input.driver="com.mysql.cj.jdbc.Driver" \
+--templateProperty jdbctojdbc.input.table="employees" \
+--templateProperty jdbctojdbc.output.url="jdbc:sqlserver://1.1.1.1:1433;databaseName=test;user=sqlserver;password=password" \
+--templateProperty jdbctojdbc.output.driver="com.microsoft.sqlserver.jdbc.SQLServerDriver" \
```
[the remark this answers is not in the document]
Comment (hhasija) [reply java/src/main/java/com/google/cloud/dataproc/templates/jdbc/README.md:535]: I think so, only one example is sufficient enough to understand.

File: CHANGELOG.md
Comment (vanshaj-bhatia) [inline CHANGELOG.md:1]: Please remove this file

**Review by vanshaj-bhatia — CHANGES_REQUESTED** [review]

**Comment by vanshaj-bhatia** [comment]
@poojabasker20 I can see some changes are not related to this issue. Can you please revert those back?

File: CHANGELOG.md
Comment (poojabasker20) [reply CHANGELOG.md:1]: Done

File: java/src/main/java/com/google/cloud/dataproc/templates/jdbc/README.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (poojabasker20) [reply java/src/main/java/com/google/cloud/dataproc/templates/jdbc/README.md]: Updated README with only one example.

**Review by vanshaj-bhatia — APPROVED** [review]
LGTM!

**Pull request merged**